apps/publications/src/app/components/publications-list/publications-list.component.html

```diff
@@ -1,18 +1,16 @@
-<div [class.hide-table]="exporting" [hidden]="dataSource.filteredData.length === 0 || publications.length === 0"
+<div [hidden]="dataSource.filteredData.length === 0 || publications.length === 0"
      class="card mt-3">
   <div class="card-body">
-    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter"
+    <perun-web-apps-table-options (exportData)="exportData($event)"
```

Comment:
Columns `Reported by` and `Thanks` should be represented by names, but in exported CSV are only counts.
![image](https://user-images.githubusercontent.com/70371638/114158619-2f387180-9925-11eb-92fe-d799591193d2.png)